docs/docs/workflows/create.md

```diff
@@ -174,12 +174,71 @@ terraform apply
 The Constellation [apply step](#the-apply-step) requires the already created `constellation-config.yaml` and the `constellation-state.yaml`.
 Create the `constellation-state.yaml` using the output from the Terraform state and the `constellation-conf.yaml`:
 
+<tabs groupId="provider-output">
+<tabItem value="aws-output" label="AWS">
```

Comment:
```suggestion
<tabs groupId="csp">
<tabItem value="aws" label="AWS">
```
groupId and value names must always be the same. Then, if you switch a tab in on place, all other places are switched, too.